$OpenBSD: patch-src_pmacctd_c,v 1.1 2021/02/03 23:14:18 sthen Exp $

From abc6a2fd647aa97707acd7ae3e44785de3aea4d3 Mon Sep 17 00:00:00 2001
From: Paolo Lucente <pl+github@pmacct.net>
Date: Mon, 18 May 2020 11:56:46 +0000
Subject: [PATCH] * fix, renamed some variables and unified declaration of
 others in order to be more gcc10 friendly. Related to Issue #394

Index: src/pmacctd.c
--- src/pmacctd.c.orig
+++ src/pmacctd.c
@@ -40,9 +40,6 @@
 #endif
 #include "jhash.h"
 
-/* variables to be exported away */
-struct channels_list_entry channels_list[MAX_N_PLUGINS]; /* communication channels: core <-> plugins */
-
 /* Functions */
 void usage_daemon(char *prog_name)
 {
